### PR TITLE
✅ [`amp-wistia-player`] Add unit tests to improve test coverage.

### LINKED ISSUE #32905: Increase unit test coverage of [amp-wistia-player]
The current test coverage of this component is 42.86%.

### PR DESCRIPTION
**Pull request by rebeccanthomas**
Fixes #32905 

This changes increases code coverage in `amp-wistia-player` from 42.86% to 83.33%
(Measured using the `--coverage` flag with `gulp unit`)

### CODE REVIEW

File: extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js [hunk 1]
```diff
@@ -15,6 +15,7 @@
  */
 
 import '../amp-wistia-player';
+import * as utils from '../../../../src/dom';
```
Comment (alanorozco) [inline extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js:18]: nit: `as dom`

**Review by alanorozco — APPROVED** [review]

File: extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js [hunk 2]
```diff
@@ -43,21 +44,83 @@ describes.realWin(
...
     }
 
-    it('renders', () => {
-      return getWistiaEmbed('u8p9wq6mq8').then((wistiaEmbed) => {
-        const iframe = wistiaEmbed.querySelector('iframe');
-        expect(iframe).to.not.be.null;
-        expect(iframe.tagName).to.equal('IFRAME');
-        expect(iframe.src).to.equal(
-          'https://fast.wistia.net/embed/iframe/u8p9wq6mq8'
+    describe('rendering', async () => {
```
Comment (dmanek) [inline extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js:47]: nit: don't need `async` here (typically `async` is used with `it`)

File: extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js [hunk 3]
```diff
@@ -43,21 +44,83 @@ describes.realWin(
...
+      });
+
+      it('can pause', () => {
+        const spy = env.sandbox.spy(impl, 'sendCommand_');
+        impl.pause();
+        expect(spy).to.be.calledWith('amp-pause');
+      });
+
+      it('can mute', () => {
+        env.sandbox.spy(impl, 'sendCommand_');
```
Comment (dmanek) [inline extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js:101]: can you change this to `const spy = ...` to remain consistent with the other tests?

File: extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js [hunk 4]
```diff
@@ -43,21 +44,83 @@ describes.realWin(
...
+        impl.mute();
+        expect(impl.sendCommand_).calledWith('amp-mute');
+      });
+
+      it('can unmute', () => {
+        env.sandbox.spy(impl, 'sendCommand_');
```
Comment (dmanek) [inline extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js:107]: same here, change to `const spy = ...`

File: extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js [hunk 5]
```diff
@@ -43,21 +44,83 @@ describes.realWin(
...
+        expect(player.querySelector('iframe')).to.be.null;
+        expect(impl.iframe_).to.be.null;
+      });
     });
 
-    it('requires data-media-hashed-id', () => {
-      return getWistiaEmbed('').should.eventually.be.rejectedWith(
-        /The data-media-hashed-id attribute is required for/
-      );
+    describe('methods', async () => {
```
Comment (dmanek) [inline extensions/amp-wistia-player/0.1/test/test-amp-wistia-player.js:77]: nit: same here, remove `async`

**Pull request merged**